### PR TITLE
feat!: add new color tokens and update alerts

### PR DESCRIPTION
**Pull request by satya-achanta-venkata**
**This change:** (check at least one)

- [x] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [x] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [x] Yes
- [ ] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?

**What does this change address?**

1. Added new color tokens `color-yellow-97` & `color-green-93`.
2. Updated documents to reflect the new color tokens.

### CODE REVIEW

**Comment by daneah** [comment]
@satya-achanta-venkata I think the new alert design that uses these colors also indicates we need to add a border around it for better visual separation from the rest of the page. I think that border is the same color as the icon, based on the Figma design. Can you add that?

I also denied a couple of the changes—looks like we also need to update the error alerts to use black text (like the rest of the alerts already do).

**Comment by daneah** [comment]
@satya-achanta-venkata Looks a lot better! I think there's one more glaring fix, which is that links in error Alert also need to be updated to black, which probably needs to happen [here](https://github.com/ithaka/pharos/blob/develop/packages/pharos/src/components/alert/pharos-alert.styles.scss).

File: packages/pharos/src/components/alert/pharos-alert.wc.stories.mdx [hunk 1]
```diff
@@ -67,7 +67,7 @@ export const ErrorTemplate = (args) => html` <pharos-alert status="error">
   <p class="alert-example__content">${args.text}</p>
   <p class="alert-example__content">
     For more information,
-    <pharos-link href="#" on-background>read the documentation</pharos-link>.
+    <pharos-link href="#">read the documentation</pharos-link>.
```
[the remark this answers is not in the document]
Comment (satya-achanta-venkata) [reply packages/pharos/src/components/alert/pharos-alert.wc.stories.mdx:70]: updated both the docs. So , resolving the conversation.

File: packages/pharos/tokens/components/alert.json [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
         "info"   : { "value": "{color.Glacier blue.80.value}" },
-        "error"  : { "value": "#dc2828" },
-        "success": { "value": "#e3eed8" },
-        "warning": { "value": "#ffefc2" }
+        "error"  : { "value": "{color.Living coral.90.value}"},
+        "success": { "value": "{color.Green.93.value}" },
+        "warning": { "value": "{color.Yellow.97.value}" }
       },
       "icon": {
         "base"   : { "value": "#333333" },
```
[the remark this answers is not in the document]
Comment (satya-achanta-venkata) [reply packages/pharos/tokens/components/alert.json]: Thanks for pointing it out @Niznikr . Fixed it.

**Review by Niznikr — APPROVED** [review]

**Review by daneah — APPROVED** [review]

File: .changeset/wicked-experts-invent.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+---
+'@ithaka/pharos': major
+'@ithaka/pharos-site': major
```
Comment (Niznikr) [inline .changeset/wicked-experts-invent.md:3]: Believe for the site it would only be a `patch` change

**Comment by kelseytrabue** [comment]
@satya-achanta-venkata I have a small piece of feedback, would you be able to change the background of the info alert to use glacier-blue-90 instead of glacier-blue-80?

**Pull request merged**